addons/mail/static/src/models/thread_viewer/thread_viewer.js

```diff
@@ -95,6 +95,20 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsLoading() {
+            if (
+                this.thread && this.threadCache && this.threadCache.isLoading &&
+                !this.isShowingLoading && !this._isPreparingLoading
+            ) {
+                this._prepareLoading();
+            }
+            return false;
```

Comment:
is loading is not a real field, it's used to trigger a compute method that is responsible to call prepare loading. Prepare loading is an handling of a timeout to avoid loading flickering when the thread cache is loaded quickly.

So `is loading` is not a real field and does not have a real sense and `is showing loading` is true when thread viewer is actually showing a loader.

Reply:
if `isLoading` is not a real field and it's used to prepare loading as a reaction to some other fields, it should be named `onX`.

Reply:
I'm doing it, just didn't pushed it yet, that's why I did put it in resolved :stuck_out_tongue: